### PR TITLE
Add overload to sec hailer

### PR DESCRIPTION
**Pull request by phil235**
This is pretty much a lighter version of #10115 
- Move sec hailer gas mask into its own file
- Adding overload when spamming the sec hailer: The sec hailer breaks permanently w/o hurting the user.

### CODE REVIEW

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	var/phrase = 0	//selects which phrase to use
+	var/phrase_text = null
+	var/phrase_sound = null
+
+
+	if(cooldown < world.time - 30) // A cooldown, to stop people being jerks
+		recent_uses++
+		if(cooldown_special < world.time - 180) //A better cooldown that burns jerks
+			recent_uses = initial(recent_uses)
+
```
Comment (Aranclanos) [inline code/modules/clothing/masks/hailer.dm]: super nitpick but there can be an else here

**Comment by Falamazeer** [comment]
This is unfair grudgecode targeting security.
If this is fair then by the same scale justice helms should implode and suck you into a dark dimension where eldritch creatures want to prison rape you, with the most vivid of details describing it followed by crashing to desktop.

Just do something reasonable, reduce the size of the text, increase the cooldown, lower the volume,Cause it to break if spammed and no longer function, the flash changes seemed pretty good solution for this, use that.

 or all four, whatever.

**Comment by Aranclanos** [comment]
is that a new copypasta

**Comment by SconesC** [comment]
Still a stupid grudgecode for a nonissue

**Comment by Jalleo** [comment]
Instead of burn damage make it stop working for 20 minuites

**Comment by optimumtact** [comment]
If you unironically use the word grudgecode your opinion will be discounted into a trash bin

**Comment by Lumipharon** [comment]
Can we just like, remove the shitty spam mask? (except from DS, DS should be awful shitty spamming salt collectors)

I NEVER see them get used for some sort of 'genuine' reason. They're just annoying, like the siren hat.

**Comment by phil235** [comment]
I changed the overload to simply break the hailing system.

**Pull request merged**